ctrd/snapshot.go

```diff
@@ -13,6 +13,13 @@ import (
 
 const (
 	defaultSnapshotterName = "overlayfs"
+
+	// add label type=image in Snapshotter metadata Info.Labels to indicate the snapshot for image.
+	TypeLabelKey = "type"
+	ImageType    = "image"
+
+	// user transfers snapshot key by context, and the key is SnapshotLabelContextKey
+	SnapshotLabelContextKey = "snapshot_label_key"
```

Comment:
synced with @wangforthinker offline, maybe we can find more specific label name here.